pkg/handlers/webhookrhobsreceiver.go

```diff
@@ -168,14 +169,15 @@ func (h *WebhookRHOBSReceiverHandler) processResolvedAlert(alert template.Alert,
 			err := h.ocm.RemoveLimitedSupport(hcID, reason.ID())
 			if err != nil {
 				metrics.IncrementFailedLimitedSupportRemoved(fn.Name)
-				metrics.SetResponseMetricFailure("clusters_mgmt")
+				// Set the metric for failed limited support response from OCM
+				metrics.SetResponseMetricFailure(config.ClustersService, config.LimitedSupport, alert.Labels["alertname"])
```

Comment:
Overall looks good to me.
Metrics details suggest it should be notification name. Can we use mfn name instead `limited_support` ?

Reply:
Done.